modules/bmo-about-the-bare-metal-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_module-type: CONCEPT
+[id="bmo-about-the-bare-metal-operator_{context}"]
+= About the Bare Metal Operator
+
+Use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster. 
+ 
+The BMO uses three resources to complete these tasks: 
+
+* `BareMetalHost`
+* `HostFirmwareSettings`
+* `FirmwareSchema`
```

Comment:
There is actually also `PreprovisioningImage` which is used internally but may be nonetheless of interest for advanced users.

Reply:
I think if this is mostly internal/advanced then it's probably too low-level for the general docs, which are already pretty dense. It would probably be suited to a knowledge-base article for the specific circumstances when it is required.